vagrant/provisioning/roles/arkcase-app/tasks/main.yml

```diff
@@ -451,7 +451,7 @@
   become: yes
   java_cert:
     cert_alias: "zylab_server_cert"
-    cert_url: "{{ zylab_server_host }}"
+    cert_url: "{{ zylab_server_host | default('') }}"
```

Comment:
The `java_cert` task will fail when it runs with a blank host.  Better to add a `when` clause to this task, and run it only when zylab_server_host is defined and non-blank

Reply:
got it